src/PointwiseFunctions/Hydro/InitialData/IrrotationalBns.hpp

```diff
@@ -70,6 +103,12 @@ tnsr::I<DataType, 3> spatial_rotational_killing_vector(
     const tnsr::I<DataType, 3>& x,
     const Scalar<DataType>& local_angular_velocity_around_z,
     const Scalar<DataType>& sqrt_det_spatial_metric);
+/// @}
+
+/// @{
+/// \brief The spatial derivative of the spatial rotational killing vector
+///
+/// As above, assumes uniform rotation around the z-axis
```

Comment:
Doxygen doesn't do a great job of preserving organization, so "As above" ends up being confusing in the rendered documentation.  Give the name of the specific thing you are trying to refer to.  (I think "spatial_rotational_killing_vector".)